portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java

```diff
@@ -1211,8 +1211,15 @@ public User addUserWithWorkflow(
 			emailAddress = emailAddressGenerator.generate(companyId, userId);
 		}
 
+		long ldapServerId = -1;
```

Comment:
I chose -1 because that is the value we had before: https://github.com/liferay/liferay-portal-ee/blob/bfc128321a6485ac94431bf819948d090baffc4e/portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java#L1284

I can change it to 0 though if you'd prefer.

Reply:
If -1 is the previous, let's go with -1. Thanks